tests/file_storage/tests.py

```diff
@@ -417,7 +461,8 @@ def get_available_name(self, name, max_length=None):
         return name
 
 
-class CustomStorageTests(FileStorageTests):
```

Comment:
No tests are lost. You still continue having `FileStorageTests` inherit from `FileStorageTestsBase`.

The reason why I made this change is that every test in `FileStorageTests` was executed twice due to `CustomStorageTests` inheriting from `FileStorageTests`. When you inherit from a class having `test_` methods you essentially make those tests to be executed for both the superclass and the subclass.

The reason why `CustomStorageTests` inherits from `FileStorageTests` is to reuse it's `setUp` and `tearDown` methods. Therefore I extracted a third class that has these two methods and made both test cases inherit from that base.